packages/custom-esbuild/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@angular-builders/custom-esbuild",
+  "version": "17.0.0",
+  "description": "Custom esbuild builders for Angular build facade. Allow to modify Angular build configuration without ejecting it",
+  "main": "dist/index.js",
+  "files": [
+    "dist",
+    "builders.json"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/just-jeb/angular-builders",
+    "directory": "packages/custom-esbuild"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "author": "JeB Barabanov",
+  "license": "MIT",
+  "engines": {
+    "node": "^14.20.0 || ^16.13.0 || >=18.10.0"
+  },
+  "keywords": [
+    "cli",
+    "angular",
+    "builder",
+    "angular6",
+    "angular-cli",
+    "esbuild",
+    "custom"
+  ],
+  "scripts": {
+    "prebuild": "yarn clean",
+    "build": "yarn prebuild && tsc && ts-node ../../merge-schemes.ts",
+    "postbuild": "yarn test && yarn run e2e",
+    "test": "jest --config ../../jest-ut.config.js",
+    "e2e": "jest --config ../../jest-e2e.config.js",
+    "clean": "rimraf dist",
+    "ci": "./scripts/ci.sh"
+  },
+  "builders": "builders.json",
+  "dependencies": {
+    "@angular-devkit/architect": ">=0.1700.0 < 0.1800.0",
+    "@angular-devkit/build-angular": "^17.0.0",
+    "@angular-devkit/core": "^17.0.0",
+    "esbuild": "0.19.5",
```

Comment:
We only need it for types so it should be in `devDependencies` (in order to avoid version conflicts with Angular CLI)